src/components/UI/atoms/product-review/ProductReviewCard.js

```diff
@@ -23,7 +23,7 @@ export default function ProductReviewCard(review) {
               onClick={() => {
                 setIsLiked(!isLiked);
               }}>
-              <img src={thumb} width={12} />
+              <img src={thumb} />
```

Comment:
Always try to give dimensions to image, otherwise UI will be displaced when page loading...